db/changelog/v-2/03-test-update-data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="03" author="kentr0w" context="dev or prod">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="execution"/>
+        </preConditions>
+        <update tableName="execution">
+            <column name="sdk" value="ubuntu:latest"/>
```

Comment:
You can provide default value in th3e same `addColumn`

Reply:
Fixed